asabiris/output/sms/service.py

```
@@ -115,59 +138,77 @@ async def send(self, sms_data, tenant=None):
-                            L.warning("Failed to parse XML response: {}".format(response_body))
+                            L.warning("Failed to parse XML response: %s", response_body)
                             raise ASABIrisError(
                                 ErrorCode.SERVER_ERROR,
                                 tech_message="Failed to parse response from SMSBrana.cz.",
-                                error_i18n_key="Error occurred while sending SMS. Reason: '{{error_message}}'.",
+                                error_i18n_key=(
+                                    "Error occurred while sending SMS. "
+                                    "Reason: '{{error_message}}'."
+                                ),
                                 error_dict={"error_message": custom_message}
                             )
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)